src/app/password/new-password-and-confirm/new-password-and-confirm.component.ts

```diff
@@ -0,0 +1,58 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { FormControl, FormGroup, ValidationErrors, Validators } from '@angular/forms';
+
+@Component({
+  selector: 'new-password-and-confirm',
+  templateUrl: './new-password-and-confirm.component.html',
+  styleUrls: ['./new-password-and-confirm.component.scss']
+})
+export class NewPasswordAndConfirmComponent implements OnInit {
+  static readonly CONFIRM_NEW_PASSWORD_FORM_CONTROL_NAME: string = 'confirmNewPassword';
+  static readonly NEW_PASSWORD_FORM_CONTROL_NAME: string = 'newPassword';
```

Comment:
Is there any way to reduce these variable name lengths? They cause line wrapping